myscript.js

```diff
@@ -0,0 +1,21 @@
+
+var dataArray = [1, 8, 7, [ 4, 65, 1, [ 2, 7 ], 45, [ 6, 4], 6 ], 4, 8]
+
+var jenyaFlatternArray = function (Array) {
+    var flattering = function (mlArray) {
+        for (var i = 0; i < mlArray.length; i++) {
+            var check = typeof mlArray[i];
```

Comment:
wow, brilliant mention of 'DRY'!!!!! you are absolutely right! believe me, authors of some huge applicates don't know this rule